extension/event-listener-sysout/src/test/java/org/keycloak/quickstart/event/listener/ArquillianSysoutEventListenerProviderTest.java

```diff
@@ -89,7 +89,7 @@ public void setup() throws IOException {
         webDriver.manage().timeouts().pageLoadTimeout(30, TimeUnit.SECONDS);
         webDriver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
 
-        String basedir = System.getProperty("basedir");
+        String basedir = System.getProperty("basedir", "/pedroigor/development/workspace/redhat/keycloak/keycloak-quickstarts/extension/event-listener-sysout");
```

Comment:
I guess you kept this line in this PR by accident? :-)

Reply:
Ok